packaging/Dockerfile.essential

```diff
@@ -0,0 +1,52 @@
+ARG DPDK_VERSION=18.05
+ARG OS_VER=18.10
+FROM ubuntu:${OS_VER}
+
+RUN apt-get update && apt-get install -y \
+    g++-5 \
+    gcc-5 \
+	libnuma-dev \
+    linux-headers-$(uname -r) \
+    autoconf \
+    make \
+    automake \
+    ethtool \
+    net-tools \
+    wget \
+    unzip \
+    tar \
+    tree \
+    git \
+    libunwind8 \
+    apt-transport-https \
+    libtool \
+    gawk \
+    libc-ares-dev \
+    python \
+    libz-dev \
+    golang-go \
+    curl \
+    && rm -rf /var/lib/apt/lists/* 
+
+RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-5 10  
```

Comment:
@fbalicchia What's the idea behind priority 10 here? How does that work? I am not very familiar with update-alternatives

Reply:
Ubuntu 18.10 provied out of the box gcc-7, instead for our case we need gcc-5. In this manner we force OS to use gcc5